library/x509_create.c

```diff
@@ -37,44 +37,46 @@ typedef struct {
     const char *name;
```

Comment:
Pre-existing: Even though it's internal, this structure should be properly documented. E.g.:

```
/* Structure linking OIDs for X.509 DN AttributeTypes to their
 * string representations and default string encodings used by Mbed TLS. */
typedef struct {
   const char *name; /* String representation of AttributeType, e.g.
                      * "CN" or "emailAddress". */
   size_t name_len;  /* Length of \c name, without trailing \c 0 byte. */
   const char *oid;  /* String representation of OID of AttributeType,
                      * as per RFC 5280, Appendix A.1. */
   int default_tag;  /* The default character encoding used for the
                      * given attribute type, e.g. 
                      * #MBEDTLS_ASN1_UTF8_STRING for UTF-8. */
}; 
```